scripts/build.sh

```diff
@@ -172,7 +172,7 @@ if $ci; then
     fi
   fi
 
-  "$VcpkgExe" install freetype glad glm lua sdl2 sdl2-image sdl2-mixer[core,mpg123,libvorbis]
+  "$VcpkgExe" install freetype glad glm lua sdl2 sdl2-image sdl2-mixer[core,mpg123,libvorbis] spdlog
```

Comment:
Same for here, looks to be the .sh version of what I asked about in the ps1 file.

Reply:
This is so that the automated builds properly build with spdlog, otherwise it would not be pulled in and compilation would fail. - the linux version